examples/quickstart/run.py

```diff
@@ -66,31 +66,29 @@ def main():
         training_data_loader=training_data_loader(),
         skew_comparison=skew_comparison(),
         drift_detector=drift_detector,
-    ).run(run_name="first_inference_pipeline_run")
+    ).run()
 
     # fetch latest runs for each pipeline
     repo = Repository()
     train_run = repo.get_pipeline(pipeline_name="training_pipeline").runs[-1]
-    inference_run = repo.get_pipeline(pipeline_name="inference_pipeline").runs[
-        -1
-    ]
+    inf_run = repo.get_pipeline(pipeline_name="inference_pipeline").runs[-1]
 
     # visualize training pipeline
     PipelineRunLineageVisualizer().visualize(train_run)
 
     # visualize inference pipeline
-    PipelineRunLineageVisualizer().visualize(inference_run)
+    PipelineRunLineageVisualizer().visualize(inf_run)
```

Comment:
what was this about?

Reply:
Just making L74 a bit more readable by reducing it to <80 chars 😄